mozetl/taar/taar_locale.py

```diff
@@ -37,26 +36,33 @@ def get_addons(spark):
     get_addons function.
     """
     return spark.sql("""
-        WITH sample AS (
-        SELECT client_id,
-        settings[0].locale AS locality,
-        EXPLODE(active_addons[0])
-        FROM longitudinal
-        WHERE normalized_channel='release'
-          AND build IS NOT NULL
-          AND build[0].application_name='Firefox'
+        WITH sample AS
```

Comment:
nit: this should probably be indented with the lower `(`

Reply:
Doh. I've pushed a commit to clean up that whitespace.